strapi/strapi
# ML: Refactor AssetMeta into ContextInfo to reuse it for folders

### What does it do?

Refactors the `AssetMeta` component to be a global component in the media library so that it can be reused for folders.

<img width="855" alt="Screenshot 2022-04-12 at 17 28 11" src="https://user-images.githubusercontent.com/2244375/162998373-91905e5b-0d2e-4b5a-9b35-ac6ee43b87d2.png">

### Why is it needed?

To implement the same context information for folders.

### How to test it?

1. Edit an asset in the media library
2. See the asset metadata displayed as before
